.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+
+node_js:
+  - 6
+  - 4
```

Comment:
Correct me if I'm wrong, but eslint will check the file statically without ever executing any code so even if eslint is running under node 4.x and returning that the code is flawless does not necessarily mean that it will work for v4.x.

What we really want is something like: https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features.md